packages/common/src/index.ts

```diff
@@ -355,3 +356,56 @@ export const getUTCDate = (date: Date): string => {
 export const getUTCDateTime = (date: Date): string => {
   return `${getUTCDate(date)} ${getUTCTime(date)}`;
 };
+
+export const RedstoneTokenIds: Record<string, string> = {
+  FTM: "FTM",
+  BUSD: "BUSD",
+  DAI: "DAI",
+  ETH: "ETH",
+};
+
+export const useTokenPrice = (tokenId: string|undefined) => {
+  const [tokenPrice, setTokenPrice] = useState<number>();
+  const [error, setError] = useState<Response | undefined>();
+  const [loading, setLoading] = useState(false);
+
+  if (!tokenId) return {
+    data: 0,
+    error,
+    loading,
+  };
+
+  useMemo(async () => {
+    setLoading(true);
+
+    const tokenPriceEndpoint = `https://api.redstone.finance/prices?symbol=${tokenId}&provider=redstone&limit=1`;
```

Comment:
should this be an env?

Reply:
That URL doesn't change. Hence didn't add it as an env